code/src/components/SubHeader.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { MdDelete } from "react-icons/md";
+import { RiCheckboxMultipleFill } from "react-icons/ri";
+import todos from "../reducers/todos";
+
+import {
+  Paragraph,
+  SendButton,
+  OptionsContainer
+} from "../styled/StyledComponent";
```

Comment:
I like how you work with styled components and that you are importing them instead of writing them directly in the component. Looks much cleaner and it is easy to follow.